src/coreclr/src/pal/src/misc/jitsupport.cpp

```diff
@@ -50,8 +50,8 @@ PAL_GetJitCpuCapabilityFlags(CORJIT_FLAGS *flags)
 //        CPUCompileFlags.Set(CORJIT_FLAGS::CORJIT_FLAG_HAS_ARM64_DCPOP);
 #endif
 #ifdef HWCAP_ASIMDDP
-//    if (hwCap & HWCAP_ASIMDDP)
-//        CPUCompileFlags.Set(CORJIT_FLAGS::CORJIT_FLAG_HAS_ARM64_DP);
+    if (hwCap & HWCAP_ASIMDDP)
```

Comment:
We should add a tracking issue with the `tracking-external-issue` label for when Windows support is in, so we can update the codeman checks for Windows.

Reply:
Done https://github.com/dotnet/runtime/issues/39094